cryptoapi_windows.go

```diff
@@ -9,16 +9,66 @@ import (
 	"time"
 
 	"golang.org/x/sys/windows/registry"
+	"gopkg.in/hlandau/easyconfig.v1/cflag"
+)
+
+var (
+	cryptoApiFlagGroup            = cflag.NewGroup(flagGroup, "capi")
+	cryptoApiFlagLogicalStoreName = cflag.String(cryptoApiFlagGroup, "store", "Root",
+		"Name of CryptoAPI logical store to inject certificate into. Consider: Root, Trust, CA, My")
+	cryptoApiFlagStoreScope = cflag.String(cryptoApiFlagGroup, "scope", "system",
+		"Scope of CryptoAPI certificate store. Valid choices: current-user, system, enterprise, group-policy")
```

Comment:
The term "scope" to refer to a physical store is a bit confusing, as (to my knowledge) Microsoft docs do not use this term, and conceptually the logical store and certain cert properties (e.g. the EKU constraints) could also be considered to constitute "scoping".  Maybe rename to `cryptoApiFlagPhysicalStoreName`?